src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -1162,7 +1162,6 @@ machine(MachineType:TCC, "TCC Cache")
   transition(I, WrVicBlkBack, W) {TagArrayRead, TagArrayWrite, DataArrayWrite} {
```

Comment:
Does this still need to do a tag array write?

Reply:
In my opinion yes -- because a_allocateBlock is updating the tag.